### DISCUSSION TITLE
mamba 32bit binaries for Windows

### DISCUSSION THREAD

**Discussion by kadykov** [conda-forge/miniforge#468] [Ideas]
Thank you for the great package manager! I always enjoy working with it, especially when I need to recreate a complex environment, `mamba` does it in tens of seconds, while `conda` in tens of minutes (if it is not stuck)...

However, from time to time I need to use 32bit Python, for example, to connect to some proprietary 32-bit libraries, and I have to switch back to `miniconda` instead of `mamba` because there is no 32bit version of mamba distribution on Windows.

Is it possible to distribute a 32-bit version of 'mamba' for Windows alongside the 64-bit?

**Comment by jaimergp** [comment]
I think you can create 32-bit environments from a regular mambaforge installation. The trick is to use export `CONDA_SUBDIR=win-32` before you run `mamba env create` or similar. I don't know how the availability of packages for `win-32` is though because it has been deprecated for a while.
